content/concepts/the-rippled-server/online-deletion.md

```diff
@@ -0,0 +1,126 @@
+
+- **`[ledger_history]`** - Specify a number of validated ledgers, equal to or less than `online_delete`. If the server does not have at least this many validated ledger versions, it attempts to backfill them by fetching the data from peers.
+
+    The default for this setting is 256 ledgers.
+
+    The following diagram shows the relationship between `online_delete` and `ledger_history` settings:
+
+    ![Ledgers older than `online_delete` are automatically deleted. Ledgers newer than `ledger_history` are backfilled. Ledgers in between are kept if available but not backfilled](img/online_delete-vs-ledger_history.png)
+
+- **`advisory_delete`** - If enabled, online deletion is not scheduled automatically. Instead, an administrator must manually trigger online deletion. Use the value `0` for disabled or `1` for enabled.
+
+    This setting is disabled by default.
+
+- **`[fetch_depth]`** - Specify a number of ledger versions. The server does not accept fetch requests from peers for historical data that is older than the specified number of ledger versions. Specify the value `full` to serve any available data to peers.
+
+    The default for `fetch_depth` is `full` (serve all available data).
+
+    The `fetch_depth` setting cannot be higher than `online_delete` if both are specified. If `fetch_depth` is set higher, the server treats it as equal to `online_delete` instead.
+
+    The following diagram shows how fetch_depth works:
+
+    ![Ledger versions older than fetch_depth are not served to peers](img/fetch_depth.png)
+
+For estimates of how much disk space is required to store different amounts of history, see [Capacity Planning](capacity-planning.html#historical-data).
+
+### Advisory Deletion
+
+By default, online deletion happens automatically and periodically. If the `advisory_delete` setting is enabled, online deletion only happens when an administrator triggers it using the [can_delete method][]. You can run this command with a scheduled job to trigger automatic deletion based on clock time instead of the number of ledger versions closed.
+
+If your server is heavily used, the extra load from online deletion can cause your server to fall behind and temporarily de-sync from the consensus network. If this is the case, you can use advisory delete and schedule online deletion to happen only during off-peak times.
+
+You can also use advisory delete for other reasons, such as if you want to manually confirm that the transaction data is backed up to a separate server before deleting it.
+
+The `can_delete` API method can enable advisory deletion (with the value `never`) or disable advisory delete (with the value `always`). These setting changes persist until you restart the `rippled` server, at which point the settings in the config file override them.
```

Comment:
They're persisted.
never: don't do online_delete
always: do online_delete as soon as the online_delete number of ledgers has been retained.
now: do online_delete at the next ledger close or as soon as the online_delete number of ledgers have been acquired, whichever is greater.
<integer> or <ledger hash>: Delete up to and including that ledger as soon as at least the online_delete number of ledgers have been acquired.

^^
What I just described is documented at: https://developers.ripple.com/can_delete.html